Graphics/UI/Gtk/WebKit/WebView.chs

```diff
@@ -767,37 +774,35 @@ webViewGetPasteTargetList = maybePeek mkTargetList <=< {#call web_view_get_paste
 
 -- * Attibutes
 
+#if WEBKIT_CHECK_VERSION(1,0,1)
 -- | Zoom level of the 'WebView' instance
 webViewZoomLevel :: WebViewClass self => Attr self Float
-webViewZoomLevel = newAttr
-   webViewGetZoomLevel
-   webViewSetZoomLevel
+webViewZoomLevel = newAttrFromFloatProperty "zoom-level"
```

Comment:
The problem with using the direct getter/setter is that it generates an [attribute][1] that is not a [property] (in the glib's sense). And only properties can trigger notifications through `Graphics.UI.Gtk.Abstract.Object.notifyProperty`, so it looks like the overhead is necessary to get this feature.

[1]: https://hackage.haskell.org/package/glib-0.13.1.0/docs/System-Glib-Attributes.html
[2]: https://hackage.haskell.org/package/glib-0.13.1.0/docs/System-Glib-Properties.html